container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/engine_functions/create_engine.go

```diff
@@ -134,6 +136,16 @@ func CreateEngine(
 		)
 	}
 
+	restAPIPortSpec, err := port_spec.NewPortSpec(uint16(restAPIPort), consts.EngineTransportProtocol, consts.HttpApplicationProtocol, defaultWait)
+	if err != nil {
+		return nil, stacktrace.Propagate(
+			err,
+			"An error occurred creating the Enclave Manager UI's http port spec object using number '%v' and protocol '%v'",
```

Comment:
is this the enclave manager or the engine's rest api?

Reply:
yes!